### PR TITLE
SAK-40322 More improvements over the top bar (quick links and username)

### PR DESCRIPTION
**Pull request by josecebe**

### CODE REVIEW

**Comment by josecebe** [comment]
Improvements over "Quick Links" box.

Before:
quicklinks_before

After:
quicklinks_after

File: library/src/morpheus-master/sass/modules/navigation/_base.scss [hunk 1]
```diff
@@ -21,7 +21,6 @@ body.is-logged-out{
 
 	.#{$namespace}headerLogo{
 		width: 100%;
-		overflow: hidden;
```
Comment (josecebe) [inline library/src/morpheus-master/sass/modules/navigation/_base.scss]: @fostersdesign do you think this overflow hidden is useless? I needed to remove it to show the "Quick Links" box after I've moved `<ul class="Mrphs-quickLinksNav__subnav">` element into `<li class="Mrphs-quickLinksNav__popup">`

Comment (fostersdesign) [reply library/src/morpheus-master/sass/modules/navigation/_base.scss]: I think the "overflow: hidden" is less useful now that we've flexed most of the containers. So, yes, I think it should be okay to remove it.

**Comment by josecebe** [comment]
The issue was resolved in the first commit, but I had a doubt (overflow: hidden) that was clarified by @fostersdesign

File: library/src/morpheus-master/sass/modules/quicklinks/_base.scss [hunk 2]
```diff
@@ -37,13 +36,15 @@
     background-color: $tool-menu-background-color;
     box-shadow: 5px 0 50px 0px rgba(0,0,0,0.5);
     position: absolute;
-    top: 45px;
-    right: 170px;
+    top: 40px;
+    right: 0;
     z-index: 99;
-    padding: 0 1em 1em 1em;
+    padding: 0.5em 1em 1em 1em;
```
Comment (fostersdesign) [inline library/src/morpheus-master/sass/modules/quicklinks/_base.scss]: This padding could be changed to:

`padding: $standard-spacing;`

File: library/src/morpheus-master/sass/modules/quicklinks/_base.scss [hunk 3]
```diff
@@ -55,51 +56,61 @@
...
       border-bottom: 1px solid $tool-border-color;
-      padding: 0 0.3em 0 0;
+      padding: 0 0 8px 0;
+      margin-bottom: 8px;
       background-color: $tool-menu-background-color;
+      line-height: normal;
 
       .#{$namespace}quickLinks-header--info {
-        vertical-align: middle;
+        max-width: 90%;
```
[the remark this answers is not in the document]
Comment (josecebe) [reply library/src/morpheus-master/sass/modules/quicklinks/_base.scss]: Not needed at all, thanks for reviewing it @fostersdesign ! I've made all changes you suggested.

**Pull request merged**